recipes/6s/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "sixs" %}
```

Comment:
Please change the directory name to reflect this one. Otherwise the feedstocking process will break. (We are fixing this BTW.)